controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/deployment/TesterCloud.java

```diff
@@ -29,14 +32,11 @@
     /** Returns whether the test container is ready to serve */
     boolean testerReady(DeploymentId deploymentId);
 
-    /** Returns whether the given URL is registered in DNS. */
-    boolean exists(URI endpointUrl);
+    /** Returns the IP address of the given host name, if any. */
+    Optional<String> resolveHostName(HostName hostname);
 
-    /**
-     * Returns whether the given URL is registered in DNS. Always returns true,
-     * as endpoints are not use in this case
-     */
-    default boolean exists(DeploymentId deploymentId) { return true; }
+    /** Returns the host name of the given CNAME, if any. */
+    Optional<HostName> resolveCName(HostName hostName);
```

Comment:
Consider renaming to either `resolveCNAME` or `resolveCname` (I think we mostly use the latter: camel-case regardless of actual casing).

Reply:
Will do the latter.